README.md

```diff
@@ -25,26 +24,39 @@ Players must locate batteries to keep their lights on, or face certain death in
 
 ### Multiplayer Networking
 
-DESCRIPTIONHERE
-    
+Our multiplayer networking implementation will synchronize the actions of up to 4 players across the network.
+One client player will also assume the role of the host. Clients will transmit actions to the server, which
+will resolve conflicts, simulate the world, and send state updates.
+
 ### Lighting Systems
 
-DESCRIPTIONHERE
+Dynamic 2D lighting will be implemented to provide a more interesting game atmosphere and add an element of
+resource/vision management to the mastery curve. Moving light sources, different light colors and color 
+blending, and different types of light sources (point, spotlight) will be implemented.
```

Comment:
Need more details on what specifically you're aiming to achieve. The content in your presentation was more than sufficient. Add more of those details here to clarify what, specifically, will count for your lighting adv topic.